package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@jupiterone/jupiterone-client-nodejs",
+  "version": "1.0.0",
```

Comment:
I think it would be good to make this a 0.1.0 release to make the statement that we expect the APIs to change. I imagine that as other projects begin using the library, and as tests are added, it will change quite a bit.